src/openforms/registrations/contrib/objects_api/templatetags/registrations/contrib/objects_api/json_tags.py

```diff
@@ -32,3 +38,22 @@ def json_summary(context):
         json_data = html_escape_json(json_data)
 
     return SafeString(json.dumps(json_data))
+
+
+@register.simple_tag(takes_context=True)
+def as_json(
+    context: template.Context, glom_path: str, default: JSONValue | JSONObject = None
+) -> str:
+    """Output the dotted path lookup on the context as a JSON string.
+
+    Example usage:
+        {% as_json "variables.kaart" default=... %}  # Will output [52.3766727, 4.8877068]
+    """
+
+    # glom uses `getattr` by default which doesn't play well with `Context`:
+    glommer = Glommer()
+    glommer.register(template.Context, get=lambda t, name: t.get(name))
+
+    data = glommer.glom(context, glom_path, default=default)
```

Comment:
there's no reason to use `glom` here, instead you can directly use Django's template mechanisms and it'll resolve the template variables during rendering.